src/components/Personalization/createComponent.js

```diff
@@ -47,6 +47,7 @@ export default ({
         const scopes = getDecisionScopes(renderDecisions, decisionScopes);
 
         if (!hasScopes(scopes)) {
+          showContainers();
```

Comment:
Isn't there an implicit "__view__" scope?  How would a user just get the "__view__" scope and render those decisions?

Reply:
yes, `getDecisionScopes(renderDecisions, decisionScopes);` if renderDecisions is `true` we add `__view__` scope in `scopes` array.

Reply:
ah, I see.